src/main/resources/edu/cornell/kfs/fp/businessobject/datadictionary/ScheduledSourceAccountingLine.xml

```diff
@@ -0,0 +1,390 @@
+<?xml version="1.0" encoding="UTF-8"?><beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:p="http://www.springframework.org/schema/p" xsi:schemaLocation="http://www.springframework.org/schema/beans         http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
+<!--
+   - The Kuali Financial System, a comprehensive financial management system for higher education.
+   - 
+   - Copyright 2005-2014 The Kuali Foundation
+   - 
+   - This program is free software: you can redistribute it and/or modify
+   - it under the terms of the GNU Affero General Public License as
+   - published by the Free Software Foundation, either version 3 of the
+   - License, or (at your option) any later version.
+   - 
+   - This program is distributed in the hope that it will be useful,
+   - but WITHOUT ANY WARRANTY; without even the implied warranty of
+   - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   - GNU Affero General Public License for more details.
+   - 
+   - You should have received a copy of the GNU Affero General Public License
+   - along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ -->
+ 
+  <bean id="ScheduledSourceAccountingLine" parent="ScheduledSourceAccountingLine-parentBean"/>
+  <bean id="ScheduledSourceAccountingLine-parentBean" abstract="true" parent="BusinessObjectEntry">
```

Comment:
Would it be possible to leverage inheritance here, maybe by having the parent be "SourceAccountingLine-parentBean to avoid duplicating already existing beans? Something like the IntraAccountAdjustmentSourceAccountingLine is doing?